tests/lib/controllers/texts-controller.test.js

```diff
@@ -0,0 +1,170 @@
+/* eslint-env jest */
+/* eslint-disable no-unused-vars */
+
+import { shallowMount, mount, createLocalVue } from '@vue/test-utils'
+import TextsController from '@/lib/controllers/texts-controller.js'
+import Alignment from '@/lib/data/alignment'
+import UploadController from '@/lib/controllers/upload-controller.js'
+import DownloadController from '@/lib/controllers/download-controller.js'
+import SourceText from '@/lib/data/source-text'
+import AppController from '@/lib/controllers/app-controller.js'
+
+describe('texts-controller.test.js', () => {
+
+  console.error = function () {}
+  console.log = function () {}
+  console.warn = function () {}
+
+  beforeAll(() => {
+    const appC = new AppController({
+      appId: 'alpheios-alignment-editor'
+    })
+    appC.defineL10Support()
+  })
+
+  beforeEach(() => {
+    jest.clearAllMocks()
+    jest.spyOn(console, 'error')
+    jest.spyOn(console, 'log')
+    jest.spyOn(console, 'warn')
+  })
+
+  it('1 TextsController - createAlignment defines alignment object ', () => {
+    const textsC = new TextsController()
+
+    textsC.createAlignment(null, null)
+
+    expect(textsC.alignment).toBeInstanceOf(Alignment)
+  })
+
+  it('2 TextsController - updateOriginDocSource creates alignment if it is not created yet ', () => {
+    const textsC = new TextsController()
+
+    textsC.createAlignment = jest.fn()
+    const docSource = { text: 'originDocSource', direction: 'ltr', lang: 'lat' }
+    textsC.updateOriginDocSource(docSource)
+
+    expect(textsC.alignment).not.toBeDefined()

```

Comment:
wouldn't we want to test that the alignment had actually been created?

Reply:
test is updated